.circleci/config.yml

```diff
@@ -113,6 +204,33 @@ jobs:
           name: Install staticcheck
           command: |
             GO111MODULE=off go get -u honnef.co/go/tools/cmd/staticcheck
+      - setup_remote_docker:
+          version: 20.10.2
+          docker_layer_caching: true
+      - run:
+          name: Install xgo
+          command: |
+            go install -v github.com/pastelnetwork/xgo@48be1f35b934924c998997a8470ffdb6a022b968
+      - create-sources-container:
+          containerName: "sourcesContainer"
+      - build:
```

Comment:
Are you sure we need to build binaries in the test job? Because without it, checking all modules takes up to 10 minutes.

Reply:
I think it is better to wait than suddenly find that Windows or MacOS build is broken

Reply:
When we release apps, it will be built anyway. To be honest, even now it takes a lot of patience to wait until all modules are tested :) And when we add more tests, it will take her more time. Every time we commit, we run this flywheel. It eats our credits on paid plan

Reply:
@levkoburburas I believe it worth it; it just caught compilation error on Windows again.
New docker image pulling time is ~ 2m 20 seconds. Also 2m 30 seconds takes compilation of pqsignatures and walletnode for macos and windows; others projects and platforms are not cross-compiled at the moment.
I'm supposing that pqsignatures cross-compilation will be removed from ci once we will have legroast integrated into walletnode project.

Reply:
Sure Max, if you say it's worth, let's try to work with it.